skypy/halos/tests/test_sampler.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import colossus as colossus
+from colossus.lss import mass_function
+from scipy import integrate
+from scipy.stats import kstest
+import pytest
+
+
+@pytest.mark.flaky
```

Comment:
```suggestion
@pytest.mark.skipif(not HAS_COLOSSUS, reason='test requires colossus')
@pytest.mark.flaky
```
Only run the test if colossus is installed (i.e. skip if `HAS_COLOSSUS==False)